app/assets/javascripts/client/views/consensus/header.html

```diff
@@ -16,15 +16,10 @@
     <div class="media">
       <div class="media-body">
           <h2>Instructions</h2>
-          <p class='instructions-content'>
-            The Readiness Assessment consists of questions across the 8 categories of the <readiness-assessment-modal title="PD System Map"></readiness-assessment-modal>. As the assessment facilitator, use this page to guide and document the conversation during the in-person consensus meeting.
-          </p>
-          <p class='instructions-content'>
-            For each question, discuss the individual responses as a group to collectively decide a consensus score, including notes for reference. You may sort or skip questions to support a targeted discussion. Save each completed response as you move through the consensus meeting.
-          </p>
+          <p class='instructions-content' ng-repeat='paragraph in instructionParagraphs()' bind-html-compile='paragraph'></p>
```

Comment:
I don't agree. Conditionals are always harder to understand than a linear flow, and as such hiding them away or disguising them makes the code more readable. The reader of this piece of code doesn't have to think "ok, so if the context is an assessment, these paragraphs are rendered; if it's an analysis, these others are"; they think "ok, so some instruction paragraphs are rendered here".
That said, this opinion isn't too strong, and I am happy to make the change if you insist.